README.md

```diff
@@ -23,32 +23,15 @@ It also requires a valid `[API_KEY]`, which authenticates requests to the API. Y
 You can find your `[API_KEY]` and your `[HOST_IDENTIFIER]` within the [Credentials](https://app.swiftype.com/as/credentials) menu:
 
 ```ruby
-client = SwiftypeAppSearch::Client.new(:host_identifier => 'host-c5s2mj', :api_key => 'api-mu75psc5egt9ppzuycnc2mc3')
+client = SwiftypeAppSearch::Client.new(:account_host_key => 'host-c5s2mj', :api_key => 'private-mu75psc5egt9ppzuycnc2mc3')
```

Comment:
This was actually updated recently to prefer `host_identifier` over `account_host_key`, which is how we are now referring this in documentation: https://github.com/swiftype/swiftype-app-search-ruby/commit/7abfa8c4878e14a5ebf05b4e5e1b32b648e8f098. Both are still supported, however.

Reply:
@JasonStoltz I got an error that claimed I needed `account_host_key`

Reply:
@JasonStoltz I'll change it back if you insist it works, I just spent a lot of time earlier trying to get it working and found these settings got it to work

Reply:
@a17levine What version of the gem do you have? If you have an older version than `0.1.4`, then that would be the case.

Reply:
I'll double check on my end as well.